api/nakadi-event-bus-api.yaml

```diff
@@ -474,7 +859,12 @@ definitions:
         type: string
       offset:
         type: string
-  SimpleStreamEvent:
+  EventStreamBatch:
+    description: |
+      One chunk of events in a stream. A batch consists of an array of `Event`s plus a `Cursor` pointing to
+      the offset of the last Event in the stream.
```

Comment:
So the `Cursor` is pointing on the last `Event` in `EventStreamBatch.events` array.